src/frontend/js/modules/avatar.js

```diff
@@ -426,13 +426,12 @@
         }
       };
     })
-    .directive('avatarPicker', function(selectionService, $alert, avatarDefaultUrl) {
+    .directive('avatarPicker', function(selectionService, $alert, avatarDefaultUrl, httpConfigurer) {
       function link($scope, element, attrs) {
         $scope.image = {
           selected: false
         };
-        $scope.avatarPlaceholder = attrs.avatarPlaceholder ? attrs.avatarPlaceholder : avatarDefaultUrl.get();
-
+        $scope.avatarPlaceholder = attrs.avatarPlaceholder ? httpConfigurer.getUrl(attrs.avatarPlaceholder) : avatarDefaultUrl.get();
```

Comment:
I don't think it's safe to request URLs provided from html ( or a user tempering HTML) with httpconfigurer ( which has headers and tokens etc)